render/egl.c

```diff
@@ -248,28 +265,28 @@ struct wlr_egl *wlr_egl_create(EGLenum platform, void *remote_display,
 			"eglUnbindWaylandDisplayWL");
 		load_egl_proc(&egl->procs.eglQueryWaylandBufferWL,
 			"eglQueryWaylandBufferWL");
+	} else {
+		if (platform == EGL_PLATFORM_DEVICE_EXT) {
+			wlr_log(WLR_ERROR, "EGL_WL_bind_wayland_display not supported");
```

Comment:
It would probably be more appropriate to not log an error when this happens. Maybe an INFO which says something like "EGL_WL_bind_wayland_display not supported on device platform, falling back to GBM platform"?

Reply:
Do you prefer INFO or DEBUG?

Reply:
As you prefer, I don't feel strongly either way. FWIW, INFO are user-visible messages, DEBUG are only visible to developers.

Reply:
Can go for INFO then